### DISCUSSION TITLE
无法被控制

### DISCUSSION THREAD

**Discussion by HaroMeng** [ZiHaoSaMa66/OsEasy-ToolBox#45] [General]
按照步骤安装配置好以后无法被教师端控制了

**Comment by ZiHaoSaMa66** [comment]
具体表现为什么原因呢
如果是找不到学生端
可以重新启动一下学生端根服务

**Comment by HaroMeng** [comment]
根服务状态<未运行&gt;是正常的吗？


运行环境：
我用的是自己的windows11电脑（24h），
华硕天选3


情况&amp;描述：
可以正常运行，并且我可以在教师机看到我自己的电脑，但是我无法用教师机控制我的电脑，就算是点击“广播管理”里的选项也没有作用。


我感觉学校的电脑太卡了，而且屏幕太昏暗了于是就用了自己的电脑。我之前一直用的自己的电脑，里面套一层虚拟机，让教师机控制我的虚拟机，这样老师下来转的时候我就可以切屏了，但是如果开启虚拟机的话再开个idea或者vscode让我的电脑卡爆了内存直接上95%了，偶然间发现了你做的这个应用程序。想要试试。






雨麟果故
***@***.***







&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ZiHaoSaMa66/OsEasy-ToolBox"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年12月28日(星期六) 下午2:58
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ZiHaoSaMa66/OsEasy-ToolBox] 无法被控制 (Discussion #45)





 
具体表现为什么原因呢
 如果是找不到学生端
 可以重新启动一下学生端根服务

**Comment by ZiHaoSaMa66** [comment]
根服务如果没有在运行中的话 学生端被杀后是不会自己自动重启且手动重启学生端是连不上教师机的噢

**Comment by ZiHaoSaMa66** [comment]
前段时间我还有在用广播管理页进行自由控制广播诶
“广播管理”里的选项也没有作用的话 请确认是否捕捉到正确的广播命令~
> 检查命令中的IPv4地址是不是学生机自身的v4地址
